src/scenes/AcceleratorRouter/Scoring/ScorecardView.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Box, Grid, useTheme } from '@mui/material';
+import { BusySpinner, Button } from '@terraware/web-components';
+
+import { Crumb } from 'src/components/BreadCrumbs';
+import Page from 'src/components/Page';
+import Card from 'src/components/common/Card';
+import { APP_PATHS } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { Scorecard as ScorecardType } from 'src/types/Score';
+
+import Scorecard from './Scorecard';
+import useScoreList from './useScoreList';
+
+const ScorecardView = () => {
+  const pathParams = useParams<{ projectId: string }>();
+  const projectId = Number(pathParams.projectId);
+  const { projectName, scorecards, status } = useScoreList(projectId);
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+
+  // This goes away when the view is implemented
+  // tslint:disable:no-console
+  console.log(scorecards);
+
+  const goToScoresEdit = useCallback(() => {
+    console.log('go to edit');
+  }, []);
+
+  const crumbs: Crumb[] = useMemo(
+    () => [
+      {
+        name: activeLocale ? `${strings.PROJECTS} / ${projectName}` : '',
+        to: APP_PATHS.ACCELERATOR_PROJECT.replace(':projectId', `${projectId}`),
```

Comment:
Should projects should go to the list of projects and project name to what you have here?
I merged a small change from my PR for Page to support non-hierarchical crumbs. Rebase with main and see crumbs in Voting.tsx